packages/discord.js/src/managers/ApplicationCommandManager.js

```diff
@@ -232,6 +232,7 @@ class ApplicationCommandManager extends CachedManager {
       type: command.type,
       options: command.options?.map(o => ApplicationCommand.transformOption(o)),
       default_permission: command.defaultPermission ?? command.default_permission,
+      default_member_permissions: (command.default_member_permissions ?? command.defaultMemberPermissions) ?? null
```

Comment:
ESLint is failing here because the () are redundant and you're missing a trailing comma. Furthermore, you don't need the `?? null`.

```suggestion
      default_member_permissions: command.default_member_permissions ?? command.defaultMemberPermissions,
```

Reply:
Changed the file according to your suggestions and everything works fine thanks for the help.